### PR TITLE
fixing tables and adding in identity concepts page

### PR DESCRIPTION
**Pull request by alkwa-msft**
Added in remark-gfm so we can render github style markdown tables

Identity page was pretty simple just needed to fix the tables

### CODE REVIEW

File: packages/storybook8/stories/Concepts/Identity.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Recommended Architecture
+
+![UI Library recommended architecture with client-server architecture](images/ui-library-architecture.png)
+
+Composite and Base Components are initialized using an Azure Communication Services access token. Access tokens should be procured from Azure Communication Services through a
+trusted service that you manage. See [Quickstart: Create Access Tokens](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-javascript) and [Trusted Service Tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/trusted-service-tutorial) for more information.
+
+These client libraries also require the context for the call or chat they will join. Similar to user access tokens, this context should be disseminated to clients via your own trusted service. The list below summarizes the initialization and resource management functions that you need to operationalize.
+
+| Contoso Responsibilities                                 | UI Library Responsibilities                                     |
```
Comment (JoshuaLai) [inline packages/storybook8/stories/Concepts/Identity.mdx:34]: Wrap the table in 
```
<Markdown>
  {`

`}
</Markdown>
```

Comment (JoshuaLai) [reply packages/storybook8/stories/Concepts/Identity.mdx:34]: Example in the `Chat.mdx` in the `StatefulClient/Chat` folder

Comment (alkwa-msft) [reply packages/storybook8/stories/Concepts/Identity.mdx:34]: I think using the official mitigation is more correct here. Can we get a confirmation of what style we want to use here? I think if we can use the remark dep then anyone else porting tables will render the table automatically for free

File: packages/storybook8/stories/Concepts/Identity.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta
```
Comment (prabhjot-msft) [inline packages/storybook8/stories/Concepts/Identity.mdx:3]: meta needs to be changed to `<Meta title="Concepts/Identity" />`

**Review by JoshuaLai — APPROVED** [review]

File: packages/storybook8/stories/Concepts/Identity.mdx (hunk 2 above)
Comment (alkwa-msft) [reply packages/storybook8/stories/Concepts/Identity.mdx]: cleaned it up to just be the meta tag with the title property

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**